db/migrate/20150421191213_add_column_to_company.rb

```diff
@@ -0,0 +1,6 @@
+class AddColumnToCompany < ActiveRecord::Migration
+  def change
+    add_column :companies, :hiring, :boolean
```

Comment:
Yes, Laura and I discussed and reached to this conclusion that we cannot give it status and as it talks about whether hiring or not, so made hiring(boolean)